contractsSol5/KyberMatchingEngine.sol

```diff
@@ -617,9 +617,9 @@ contract KyberMatchingEngine is KyberHintHandler, IKyberMatchingEngine, Withdraw
         }
 
         if (bestReserve.destAmount == 0) return (reserveArr[bestReserve.index], 0, false);
-        
+
         reserveCandidates[0] = bestReserve.index;
-        
```

Comment:
I thought I removed all the spaces.

its some automatic command in vs code.
how do we still have it...
:(

Anyway good u remove it.